tcs_catalog/query_tool/query_get_review_scorecard.txt

```diff
@@ -0,0 +1 @@
+select * from scorecard where scorecard_type_id = 2
```

Comment:
@skyhit I think this is wrong.  We should be pulling the active scorecards only for the current challenge type, so we'll need to pass in a param for the challenge type...